tests/TextDiffTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests;
+
+use PhpCsFixer\Console\Command\FixCommand;
+use PhpCsFixer\Report\ReporterFactory;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class TextDiffTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @param string $expected
+     * @param string $format
+     * @param bool   $isDecorated
+     *
+     * @dataProvider provideDiffReporting
+     */
+    public function testDiffReportingDecorated($expected, $format, $isDecorated)
+    {
+        $command = new FixCommand();
+        $commandTester = new CommandTester($command);
+        $commandTester->execute(
+            array(
+                'path' => array(__DIR__.'/Fixtures/FixCommand/TextDiffTestInput.php'),
```

Comment:
the tests fail because;
1 - fix is not implemented
2 - file is not checked in the first place, @see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1900